web/test/plugins/index.js

```diff
@@ -55,4 +55,16 @@ module.exports = (on, config) => {
     }
   });
 
+  on('task', {
+    'db:delete': (params) => {
+      db.collection(params.collection).get().then(function (querySnapshot) {
+        querySnapshot.forEach(function (doc) {
+          if ((doc.id).startsWith("test-")) {
+            db.collection(params.collection).doc(doc.id).delete();
+          }
+        });
+      });
+      return null;
```

Comment:
Is the `return` needed?

Reply:
Yes If I  do not return anything , it throws an error "Task should return promise or value"

Reply:
I think it would be best then to not use `{}` to open a function and just return the chain, something like...

```typescript
'db:delete': (params) => db.collection(params.collection).get()
     .then((querySnapshot) => querySnapshot.forEach((doc) => {
           if ((doc.id).startsWith("test-")) {
             db.collection(params.collection).doc(doc.id).delete();
           }
         });
       });
```

Reply:
Yes , I will make these changes